bin/local-env/install-wordpress.sh

```diff
@@ -127,6 +127,8 @@ container chmod 767 /var/www/html/favicon.ico
 # Activate Google Site Kit plugin.
 status_message "Activating Google Site Kit plugin..."
 wp plugin activate google-site-kit --quiet
+# Reset post-activate state.
+wp google-site-kit reset
```

Comment:
This fixes a potential error from showing up in the log which may or may not have caused a failure before but has been seen many times in the E2E log as part of the tagmanager setup test, but it's unrelated to that. It was only happening here because it was the first test that always runs.

This was due to compatibility checks that ran the very first time the `plugins.php` screen loaded due to our activation flag that controls the presence of the activation notification (which runs compatibility checks). To fix this, we simply reset the plugin after the initial activation. We already do a plugin reset in the bootstrap.js in the main `beforeAll` but it's too late as it's after the call to deactivate utility plugins. I thought about moving or changing this, but the above change is just simpler, and actually gives us slightly better coverage in that we actually use our reset command.